website/pages/docs/builders/ucloud-uhost.mdx

```diff
@@ -150,8 +157,59 @@ Here is a example for build UCloud Ubuntu image:
 }
 ```
 
+</Tab>
+<Tab heading="HCL2">
+
+```hcl
+// .pkr.hcl file
+variable "ucloud_public_key" {
+  type = string
+  default = "xxx"
+}
+
+variable "ucloud_private_key" {
+  type = string
+  default = "xxx"
+}
+
+variable "ucloud_project_id" {
+  type = string
+  default = "xxx"
+}
+
+variable "password" {
+  type  = string
+  default = "ucloud_2020"
+}
+
+source "ucloud-uhost" "basic-example" {
+  public_key        =  var.ucloud_public_key
+  private_key       =  var.ucloud_private_key
+  project_id        =  var.ucloud_project_id
+  region            =  "cn-bj2"
+  availability_zone =  "cn-bj2-02"
+  instance_type     =  "n-basic-2"
+  ssh_password      =  var.password
+  source_image_id   =  "uimage-irofn4"
+  ssh_username      =  "ubuntu"
+}
+
+build {
+  source "sources.ucloud-uhost.basic-example" {
+    image_name =  "packer-test-ubuntu{{timestamp}}"
```

Comment:
here, the hcl2 timestamp function can also be used 

```suggestion
    image_name =  "packer-test-ubuntu-${timestamp()}"
```

Reply:
note that the function returns something like `2018-05-13T07:44:12Z` and I'm not sure this is allowed.

Reply:
@azr Just now,  I have used your suggestion about `hcl2 timestamp` to play again, that looks great! Thanks for your suggestions.

Reply:
super nice !